pgx-tests/src/tests/schema_tests.rs

```diff
@@ -4,14 +4,58 @@ use pgx::*;
 
 #[pgx::pg_schema]
 mod test_schema {
+    use pgx::datum::sql_entity_graph::{PgxSql, SqlGraphEntity};
     use pgx::*;
     use serde::{Deserialize, Serialize};
 
     #[pg_extern]
     fn func_in_diff_schema() {}
 
+    #[pg_extern]
+    #[to_sql(false)]
+    fn func_elided_from_schema() {}
+
+    #[pg_extern]
+    #[to_sql(generate_function)]
+    fn func_generated_with_custom_sql() {}
+
     #[derive(Debug, PostgresType, Serialize, Deserialize)]
     pub struct TestType(pub u64);
+
+    #[derive(Debug, PostgresType, Serialize, Deserialize)]
+    #[to_sql(false)]
```

Comment:
As mentioned in a comment, `#[pgx(to_sql = false)]` would be be more matching how we see `serde` and similar crate do it.